lib/heborn_migration/model/heborning_user.ex

```diff
@@ -0,0 +1,44 @@
+defmodule HEBornMigration.Model.HEBorningUser do
+
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{
+    display_name: String.t,
+    token: String.t
+  }
+
+  @primary_key false
+  schema "heborning_users" do
+    field :token, :string,
+      primary_key: true
+
+    field :display_name, :string, size: 15,
+      null: false
+  end
+
+  @spec create(String.t, String.t) :: Ecto.Changeset.t
+  def create(token, display_name) do
+    changeset(%__MODULE__{}, %{token: token, display_name: display_name})
+  end
+
+  def changeset(struct, params) do
+    struct
+    |> cast(params, [:token, :display_name])
+    |> validate_required([:token, :display_name])
+    |> validate_length(:display_name, max: 15)
```

Comment:
This is based on renato's information about HE1 usernames, I don't remember HE2 limit, but if it's lower than that, it should be updated.